README.md

```diff
@@ -89,6 +89,18 @@ In order to test caching locally,
 
 To run stress testing against the Salesforce instance, refer to the documentation in the [stress testing folder](load_testing/load_testing.md)
 
+## Pulling and pushing translations from [our project on Phrase](https://app.phrase.com/accounts/city-county-of-san-francisco/projects/dahlia-sf-dahlia-web/dashboard)
+To get started working with our Phrase translations, you will need to:
+
+1. Install the Phrase CLI with `brew tap phrase/brewed && brew install phrase`
+1. [Create and save a new access token](https://app.phrase.com/settings/oauth_access_tokens)
```

Comment:
Can you explain where to store this token? I haven't created a Phrase account yet so I haven't actually read the info at that link

Reply:
In lastpass 😕

Reply:
Oh I missed the piece just under this that explains you need to pass `--phraseAccessToken=[your access token]` with each command. Okay!

Reply:
I'll update to make it clearer

Reply:
updated

Reply:
Thanks, looks good